### ISSUE TITLE
make gettext doesn't generate translation string for parametrs of function

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1417]
Hello, we are using html output from sphix for quite a while and we are extremely pleased, that it works so well. Now we wanted to have localised versions of documentation. 

```
def add_key(self, key):
    """
    :param key: new key
    :type key: str

    usecase::

          class.add_key("item")

    Normal continuing of docstring...
    """
```

So we set up gettext and used make gettext to generate messages.
It worked withouth problem, until we realised, that parameters of functions, methods and so on are not genereated. Is this how it should be? We get to messages just a last line, so something like this. Parametrs are missing.

```
msgid "Normal continuing of docstring..."
msgstr ""

msgid "Usecase::"
msgstr ""
```

Is there a easy way how to change this and append parameters? Thank you for awesome program.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1417
- Originally reported by: Anonymous
- Originally created at: 2014-03-10T07:21:33.891

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-10 06:33:00+00:00_

Hi, to clarify, the msgid "new key" is missing?

[Takayuki Shimizukawa](https://bitbucket.org/shimizukawa) can you have a look?

**Comment by shimizukawa** [comment]
_From [AlfonzM](https://bitbucket.org/AlfonzM) on 2014-03-10 07:02:11+00:00_

Hello,

yes you are right msgid "new key" is missing in generated pot file. Maybe we have some bad settings or something, but I guess that when txt output or html output works, gettext should be similiar, am I right?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-10 07:03:36+00:00_

Thanks for the confirmation. Unfortunately it's a bit more involved than that, but Takayuki-san has the most insight there and should be able to come up with a fix.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-03-12 11:31:24+00:00_

I tried to reproduce the issue, but I couldn't reproduce it. Attached sphinx project file is what I was used to operation. Please check the difference with  your project files. Thanks

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-03-12 11:37:35+00:00_

Sorry I've re-confirmed with the explicit versions.
- Sphinx-1.1.3 : the issue is reproduced. It's a limitation of the Sphinx-1.1.3.
- Sphinx-1.2.1 : the issue is not reproduced. The problem was fixed since 1.2.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-03-12 11:39:05+00:00_

[AlfonzM](https://bitbucket.org/AlfonzM) please use Sphinx-1.2.x. The issue was resolved since 1.2. Thanks.

**Comment by shimizukawa** [comment]
_From [AlfonzM](https://bitbucket.org/AlfonzM) on 2014-03-12 13:02:26+00:00_

Hello,

oh..well we use Sphinx on Ubuntu and most of computers have 12.04 and probably also version 1.1.3. We will try to upgrade, anyway thanks for this information.
